content/_data/changelogs/weekly.yml

```diff
@@ -1354,6 +1354,62 @@
     # pull: 3085 too minor (Chinese l10n)
     # pull: 3096 internal cleanup
     # pull: 3098 flaky test
+- version: "2.87"
+  date: 2017-10-29
+  changes:
+    - type: rfe
+      message: >
+        Stapler library upgraded from 1.252 to 1.253 with Servlet 3.1 support, improved Blue Ocean performance and changes of interest to plugin developers.
+      references:
+        - issue: 37062
+        - url: https://github.com/stapler/stapler/blob/master/CHANGELOG.md#1253
+          title: Stapler changelog
+      pull: 3033
+    - type: rfe
+      message: >
+        Commons Codec library upgraded from 1.8 to 1.9.
+      pull: 3033
+    - type: rfe
+      message: >
+        Agents JVM must be 1.8+ and a clear message is shown in connection logs if it is not.
+      pull: 2915
+      issue: 44851
+    - type: rfe
+      message: >
+        Major improvement to Italian localization
+      pull: 3075
+    - type: rfe
+      message: >
+        Improvements to Chinese localization
+      references:
+        - pull: 3104
+        - pull: 3105
+    - type: rfe
+      message: >
+        Retrieving the list of installed plugins now consumes much less memory.
```

Comment:
@Jimilian PTAL – is there a way to better describe the problem here?

Reply:
This is fine. Thanks you.